portafly/src/App.tsx

```diff
@@ -17,60 +19,52 @@ const navItems = [
     title: 'Analytics',
     to: '/analytics',
     items: [
-      {to: '/analytics/usage', title: 'Usage'},
+      { to: '/analytics/usage', title: 'Usage' },
     ],
   },
   {
     title: 'Applications',
     to: '/applications',
     items: [
-      {to: '/applications', title: 'Listing'},
-      {to: '/applications/plans', title: 'Application Plans'}
+      { to: '/applications', title: 'Listing' },
+      { to: '/applications/plans', title: 'Application Plans' },
```

Comment:
Personally, I don't like this `comma-dangle` rule. 
It looks ugly having unncessary commas all around. The only advantage is when you see the diff.
Is not a big deal, if you both like it I can live with it

Reply:
The diff is precisely the reason why. I don't particularly like it either.. I think you can achieve the same effect by adding items at the top instead of at the bottom.

Reply:
In that case, I think we can remove it

Reply:
OK